pyomo/contrib/appsi/solvers/gurobi.py

```diff
@@ -857,6 +857,43 @@ def _set_objective(self, obj):
         self._solver_model.setObjective(gurobi_expr + value(repn_constant), sense=sense)
         self._needs_updated = True
 
+    def _add_column(
```

Comment:
Any particular reason that `gurobi` and `highs` only have protected instantiations of this method?

Reply:
Yes, they are the two appsi solvers that inherit from `PersistentBase`. They get their public implementation from that base class.